snuba/snuba_migrations/events_analytics_platform/0041_hashed_keys_index.py

```diff
@@ -0,0 +1,46 @@
+from typing import Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+
+buckets = 40
+
+
+def get_hashed_keys_index_expression() -> str:
+    column_expressions = []
+    for i in range(buckets):
+        hashed_keys = f"arrayMap(k -> cityHash64(k), mapKeys(attributes_string_{i}))"
```

Comment:
Excluding `float` buckets?

Reply:
I think the way for the expression here is to actually make 1 column where we store all the hashes in an array and create the index on that column.

Otherwise, I don't think ClickHouse will use this index at all.